pkgs/development/tools/xc/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "xc";
+  version = "0.0.154";
+
+  src = fetchFromGitHub {
+    owner = "joerdav";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-GJBSPO0PffGdGAHofd1crEFXJi2xqgd8Vk2/g4ff+E4=";
+  };
+
+  vendorHash = "sha256-XDJdCh6P8ScSvxY55ExKgkgFQqmBaM9fMAjAioEQ0+s=";
+
+  subPackages = [ "cmd/xc" ];
```

Comment:
This also doesn't seem to be necessary, but I'm not familiar enough with Go stuff to really tell, so please disregard me if I'm wrong here.

Reply:
No I think your right, I only have one binary in there so I don't have to specify.